.swiftlint.yml

```diff
@@ -7,6 +7,10 @@ custom_rules:
     name: WGS84 Spatial Reference
     regex: AGSSpatialReference\(wkid:\s*(4326)\)
     message: "Use AGSSpatialReference.wgs84() instead."
+  closure_opening_space:
+    name: Closure Opening Space
+    regex: \{\[|\{\s{2,}\[
```

Comment:
I'm not an expert on regular expressions by any means, but might this be a little easier to understand? I think it is equivalent.

```suggestion
    regex: \{(?! )\[
```

Reply:
Mine flags `{[` as well as instances with more than one whitespace between them, such as `{   [`. In essence the rule enforces one and only one space. Not sure if there's a simpler way to write that but feel free to propose one.

Reply:
> Not sure if there's a simpler way to write that but feel free to propose one.

I did.

![screen shot 2018-11-21 at 12 56 30 pm](https://user-images.githubusercontent.com/2257493/48867985-ed76f280-ed8c-11e8-9b99-ddad529760c2.png)

Reply:
Huh. I tried it in RegexPal and it didn't match multiple spaces.

Reply:
Apparently what I came up with isn't the same, although I can't explain why. I did say I wasn't an expert!